src/nunit-gui/Model/TestModel.cs

```diff
@@ -240,8 +240,7 @@ public void UnloadTests()
             _files = null;
             _resultIndex.Clear();
 
-            if (TestUnloaded != null)
-                TestUnloaded(new TestEventArgs(TestAction.TestUnloaded));
```

Comment:
Actually, I intended to use ? operator. Good catch. I'll check other events as well.